apps/cron/src/tasks/processBuilderActivity/recordMergedPullRequest.ts

```diff
@@ -278,7 +278,9 @@ export async function recordMergedPullRequest({
               questTypes.push('contribute-celo-repo');
             }
 
-            await completeQuests(githubUser.builderId, questTypes);
+            if (questTypes.length) {
```

Comment:
would be better if completeQuests just silently continue so you dont have to add this logic here. i think i saw it someplace else also